src/Modules/Backend/Domain/Action/AbstractFormActionController.php

```diff
@@ -55,11 +55,13 @@ public function getResponse(Request $request): Response
     abstract protected function getFormResponse(Request $request): ?Response;
 
     /**
+     * @codingStandardsIgnoreStart
```

Comment:
You can ignore a file in its entirety but that would beat the point for the ci

Reply:
@carakas we could ignore that specific rule for that specific file.
But I do favour the clutter instead of ignoring that rule for the whole file in an external file. The clutter ads context in my opinion.